cristin-organization/src/test/java/no/unit/nva/cristin/organization/OrganizationKarateRunner.java

```diff
@@ -0,0 +1,14 @@
+package no.unit.nva.cristin.organization;
+
+import com.intuit.karate.junit5.Karate;
+import org.junit.jupiter.api.Tag;
+
+@Tag("karateTest")
+public class OrganizationKarateRunner {
+
+    @Karate.Test
```

Comment:
So you had to change back to Karate.Test annotation?

Reply:
Otherwise the tests will not run with karate runtime, just say it is OK without actualy runing karate